src/rfac/rfgetsa.c

```
@@ -0,0 +1,87 @@
+  if ((fp = fopen(fname_string, "r")) == NULL)
+  {
+    ERROR_MSG("");
+    perror(fname_string);
+    exit(EIO);
+  }
+
+  while (fgets(linebuffer, STRSZ, fp) != NULL)
+  {
+    if (!strncasecmp(linebuffer, "sa:", 3))
+    {
+      if (sscanf(linebuffer+3, " %u", &sa) < 1)
```

Cppcheck (2.19.0)

[portability] %u in format string (no. 1) requires 'unsigned int *' but the argument type is 'size_t * {aka unsigned long *}'.

(invalidScanfArgType_int)